Gemfile

```diff
@@ -72,3 +75,5 @@ end
 gem "httparty", "~> 0.18.0"
 
 gem "firebase", "~> 0.2.8"
+
+gem 'devise-encryptable'
```

Comment:
lo mismo con esta gema, y trata de agregar el newline at end of file siempre. Como tip, vscode o sublime (asumo que otros también) se pueden configurar para que siempre agreguen eso si falta al guardar un archivo.